dashboard/app/models/reference_guide.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: reference_guides
+#
+#  id                :bigint           not null, primary key
+#  display_name      :string(255)
+#  key               :string(255)      not null
+#  course_version_id :integer          not null
+#  content           :text(65535)
+#  order             :integer          not null
```

Comment:
if it's not too annoying... we call this "position" on 6 other db tables, so that's what I'd recommend calling it here too.